_layouts/default.html

```diff
@@ -24,9 +24,12 @@
         <meta property="og:image" content="/images/bisq-og.jpg">
         <meta property="og:site_name" content="{{ langdata.site_name }}">
         <meta property="og:type" content="website">
+        <meta property="og:url" content="https://bisq.network/">
         <meta property="twitter:title" content="{{ page.title }}">
         <meta property="twitter:card" content="summary_large_image">
-        <meta property="twitter:site" content="{{ langdata.enabled }}">
+        <meta property="twitter:site" content="https://bisq.network/">
+        <meta property="twitter:description" content="{{ langdata.site_desc }}">
+        <meta property="twitter:image" content="/images/bisq-og.jpg">
```

Comment:
> Twitter cards do not support relative paths, and you have to use a fully-qualified HTTP(S) URL in the image tag.

<img width="495" alt="Screen Shot 2020-03-10 at 3 35 23" src="https://user-images.githubusercontent.com/232186/76246173-8b434900-6280-11ea-8a20-e5dd26d8a3ae.png">

Good job "fixing" the twitter tag, did you guys even test this? lol
The twitter card preview image is broken now :)